Kernel/init.cpp

```diff
@@ -109,8 +110,8 @@ extern "C" [[noreturn]] void init()
     gdt_init();
     idt_init();
 
-    setup_interrupts();
     setup_acpi();
+    setup_interrupts();
```

Comment:
Please don't move the `setup_interrupts()` call as it will break the initialization process. We setup interrupts before ACPI to make sure that if we happen to initialize an ACPI `DynamicParser`, it will use the correct IRQ controller.